lib/graphql/schema_comparator/diff/directive_argument.rb

```diff
@@ -19,7 +19,7 @@ def diff
             changes << Changes::DirectiveArgumentDefaultChanged.new(directive, old_arg, new_arg)
           end
 
-          if old_arg.type != new_arg.type
+          if old_arg.type.graphql_definition != new_arg.type.graphql_definition
```

Comment:
Could also compare by `graphql_name` which is more compatibility if/when this gem switches to all class-based type support.

Reply:
Using `graphql_definition` was borrowed from https://github.com/xuorig/graphql-schema_comparator/blob/master/lib/graphql/schema_comparator/diff/argument.rb#L24